README.md

```diff
@@ -95,10 +95,31 @@ See [contributing documentation](./CONTRIBUTING.md)
 See [full documentation](docs/README.md)
 
 
+# Useful tips
+
+## Updating kuzzle's containers
+
+When you already have installed an old version of kuzzle, don't forget to update kuzzle's containers with :
+
+```
+    $ docker-compose -f docker-compose/test.yml pull 
+```
+
+## Updating kuzzle's dependencies 
+
+To ensure if your kuzzle's dependencies are updated, go into kuzzle's containers and run :
```

Comment:
`To ensure that Kuzzle's dependencies are up-to-date, go into a Kuzzle container and run:`